crates/ark/src/lsp/inputs/library.rs

```diff
@@ -0,0 +1,148 @@
+//
+// library.rs
+//
+// Copyright (C) 2025 by Posit Software, PBC
+//
+
+use std::collections::HashMap;
+use std::path::PathBuf;
+use std::sync::Arc;
+use std::sync::RwLock;
+
+use super::package::Package;
+use crate::lsp;
+
+/// Lazily manages a list of known R packages by name
+#[derive(Default, Clone, Debug)]
+pub struct Library {
+    /// Paths to library directories, i.e. what `base::libPaths()` returns.
+    pub library_paths: Arc<Vec<PathBuf>>,
```

Comment:
```suggestion
    pub library_paths: Vec<PathBuf>,
```

Is it overkill to Arc this? Probably like 1 or 2 items right?